### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by montjoie**
Lots of random patches, the principal patch is to fix a startup issue when users have firstname/lastname

### CODE REVIEW

File: lava-slave/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM baylibre/lava-slave-base:2018.11-1_bpo9_1
 
+RUN apt-get update
```
Comment (khilman) [inline lava-slave/Dockerfile:3]: Doesn't this need `DEBIAN_FRONTEND=noninteractive` and `-y` ?

File: README.md [hunk 2]
```diff
@@ -271,6 +271,7 @@ boards:
     type: the LAVA device-type of this device
     slave:		(optional) Name of the slave managing this device. Default to first slave found or default_slave if set.
     kvm: (For qemu only) Does the qemu could use KVM (default: no)
+    tap: (For qemu only) Does the qemu could use TAP devices (default: no)
```
Comment (khilman) [inline README.md:279]: Why is the default `no`?

**Review by khilman — CHANGES_REQUESTED** [review]

File: lava-slave/Dockerfile (hunk 1 above)
Comment (montjoie) [reply lava-slave/Dockerfile:3]: No since this command is not interactive

File: README.md (hunk 2 above)
Comment (montjoie) [reply README.md:279]: Basic qemu does not do network. Furthermore the less is mounted in docker the better it is. (security pov).

**Pull request merged**